.github/workflows/update-website-stats.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         export GITHUB_TOKEN=${{ secrets.ZAPBOT_TOKEN }}
         git config --global user.email "12745184+zapbot@users.noreply.github.com"
         git config --global user.name $GITHUB_USER
+        git config pull.rebase false
```

Comment:
I think it needs `--global` since it's not yet in a repo.

Reply:
Good point - done 😁